ModulePresentationsForCAP/gap/ModulePresentationFunctors.gi

```diff
@@ -158,3 +158,60 @@ InstallMethod( FunctorLessGeneratorsRight,
     return functor;
     
 end );
+
```

Comment:
1) What about the equivalent functor for right presentations?
2) FunctorHomLeft -> FunctorDualLeft (Hom is too general)